### DISCUSSION TITLE
Set options like tab stops by language

### DISCUSSION THREAD

**Discussion by mike-ward** [zed-industries/zed#13404] [Feature Requests]
If there a way to set tab stops for specific languages?

**Comment by omarjatoi** [comment]
You can override the `tab_size` property by adding it to the [`"languages"` section in `settings.json`](https://zed.dev/docs/configuring-zed#languages), e.g.:

```json
{
    "languages": {
        "JSON": {
            "tab_size": 4
        }
    }
}
```

I'm not sure if that's the same thing as tab stops though!

**Reply by mike-ward** [reply]
Yes, that is what I want. Thanks.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
